### ISSUE TITLE
empty maven-metadata-local.xml files

### ISSUE THREAD

**Issue by mstormi** [openhab/openhab-distro#544]
I get a large number of exceptions like the one below because there are empty (0 byte in size) files `maven-metadata-local.xml` for most bindings or karaf bundles.
The parser should accept empty files OR these at least should be removed from the distribution.

```
2017-09-26 07:38:36.372 [DEBUG] [e.karaf.kar.internal.FeatureDetector] - File 'maven-metadata-local.xml' is not a features file.
org.xml.sax.SAXParseException: Vorzeitiges Dateiende.
        at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:203) [?:?]
        at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:177) [?:?]
        at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:400) [?:?]
        at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:327) [?:?]

```

**Comment by kaikreuzer** [comment]
Not an issue on openHAB side. I have created https://issues.apache.org/jira/browse/KARAF-5394 for it.

**Comment by kaikreuzer** [comment]
Issue has been fixed in Karaf 4.1.3, which is now used in openHAB since build 1083.
